esphome/components/shelly_dimmer/shelly_dimmer.cpp

```diff
@@ -0,0 +1,490 @@
+#include "esphome/core/defines.h"
```

Comment:
For some of the below comments
```suggestion
#include "esphome/core/defines.h"
#include "esphome/core/helpers.h"
```